public/app/features/browse-dashboards/state/hooks.ts

```diff
@@ -36,6 +44,10 @@ export function useSelectedItemsState() {
   return useSelector((wholeState: StoreState) => wholeState.browseDashboards.selectedItems);
 }
 
+export function useSelectedItemsForBackendState() {
```

Comment:
naming is hard 😬 

wydt about renaming `useSelectedItemsState` and this to instead be `useCheckboxSelectionState` and `useApiSelectionState`? 🤔

Reply:
- `useCheckboxSelectionState` - sounds good 👍🏻 
 - `useAPISelectionState` - less certain on 'API', but I can't think of much better. `useActionSelectionState`? 🥴